symfony/src/Entity/PatientStatus.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\PatientStatusRepository")
+ */
+class PatientStatus
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $participantId;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $organization;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $awardee;
+
+    /**
+     * @ORM\OneToMany(targetEntity="App\Entity\PatientStatusHistory", mappedBy="patient_status")
```

Comment:
This is throwing an invalid entity mapping error. The mapped by should refer to the entity name, not the column name.

```suggestion
     * @ORM\OneToMany(targetEntity="App\Entity\PatientStatusHistory", mappedBy="patientStatus")
```

Reply:
This might have been missed while changing entity names to camel case. Will fix that. Thanks!